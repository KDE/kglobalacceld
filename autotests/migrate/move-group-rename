[kstuff]
Launch=Meta+K,Meta+K,Launch

[org.kde.foo.desktop]
_launch=none,Meta+P,Foo

[org.kde.bar.desktop]
DoSomething=Meta+X,Meta+X,Do Something
