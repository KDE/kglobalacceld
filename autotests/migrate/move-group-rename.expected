[kstuff]
Launch=Meta+K,Meta+K,Launch

[org.kde.buf.desktop]
_launch=none,Meta+P,Foo

[org.kde.bar.desktop]
DoSomething=Meta+X,Meta+X,Do Something
